FlowCrypt/App/main.swift

```diff
@@ -9,16 +9,9 @@
 import Foundation
 import UIKit
 
-autoreleasepool {
```

Comment:
Not sure we could skip `autoreleasepool` here

Reply:
I'm not sure either - the IDE was telling me that it's no longer needed. @sosnovsky may know